2024-10-28 20:06:08.862 | INFO     | aiogram.dispatcher.dispatcher:start_polling:358 - Start polling.
2024-10-28 20:06:16.772 | INFO     | aiogram.contrib.middlewares.logging:on_pre_process_message:36 - Received message [ID:78] in chat [private:6948741611]
2024-10-28 20:06:16.830 | INFO     | aiogram.contrib.middlewares.logging:on_pre_process_error:150 - Process update [ID:728620799]: [failed] (in 58 ms)
2024-10-28 20:06:16.830 | ERROR    | asyncio.base_events:default_exception_handler:1785 - Task exception was never retrieved
future: <Task finished name='Task-375' coro=<Dispatcher._process_polling_updates() done, defined at C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\dispatcher\dispatcher.py:407> exception=ChatNotFound('Chat not found')>
Traceback (most recent call last):

> File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\dispatcher\dispatcher.py", line 415, in _process_polling_updates
    for responses in itertools.chain.from_iterable(await self.process_updates(updates, fast)):
                     │         │     │                   │    │               │        └ True
                     │         │     │                   │    │               └ [<Update {"update_id": 728620799, "message": {"message_id": 78, "from": {"id": 6948741611, "is_bot": false, "first_name": "ve...
                     │         │     │                   │    └ <function Dispatcher.process_updates at 0x000001BB010F62A0>
                     │         │     │                   └ <aiogram.dispatcher.dispatcher.Dispatcher object at 0x000001BB015A2090>
                     │         │     └ <method 'from_iterable' of 'itertools.chain' objects>
                     │         └ <class 'itertools.chain'>
                     └ <module 'itertools' (built-in)>
  File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\dispatcher\dispatcher.py", line 235, in process_updates
    return await asyncio.gather(*tasks)
                 │       │       └ [<coroutine object Handler.notify at 0x000001BB0142BD80>]
                 │       └ <function gather at 0x000001BB7F8167A0>
                 └ <module 'asyncio' from 'C:\\Users\\newsp\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\asyncio\\__init__.py'>
  File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\dispatcher\handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
                     │           │        │       └ {}
                     │           │        └ (<Update {"update_id": 728620799, "message": {"message_id": 78, "from": {"id": 6948741611, "is_bot": false, "first_name": "ve...
                     │           └ <bound method Dispatcher.process_update of <aiogram.dispatcher.dispatcher.Dispatcher object at 0x000001BB015A2090>>
                     └ Handler.HandlerObj(handler=<bound method Dispatcher.process_update of <aiogram.dispatcher.dispatcher.Dispatcher object at 0x0...
  File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\dispatcher\dispatcher.py", line 256, in process_update
    return await self.message_handlers.notify(update.message)
                 │    │                │      │      └ <aiogram.types.fields.Field object at 0x000001BB00F83DD0>
                 │    │                │      └ <Update {"update_id": 728620799, "message": {"message_id": 78, "from": {"id": 6948741611, "is_bot": false, "first_name": "ven...
                 │    │                └ <function Handler.notify at 0x000001BB01024C20>
                 │    └ <aiogram.dispatcher.handler.Handler object at 0x000001BB015A20D0>
                 └ <aiogram.dispatcher.dispatcher.Dispatcher object at 0x000001BB015A2090>
  File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\dispatcher\handler.py", line 117, in notify
    response = await handler_obj.handler(*args, **partial_data)
                     │           │        │       └ {'state': <aiogram.dispatcher.storage.FSMContext object at 0x000001BB013C2ED0>}
                     │           │        └ (<Message {"message_id": 78, "from": {"id": 6948741611, "is_bot": false, "first_name": "venv∆", "username": "venvfuck", "lang...
                     │           └ <function start_message at 0x000001BB014BC680>
                     └ Handler.HandlerObj(handler=<function start_message at 0x000001BB014BC680>, spec=FullArgSpec(args=['message', 'state'], vararg...

  File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\handlers\common.py", line 173, in start_message
    await message.forward(config.links.forward_channel_id)
          │       │       │      │     └ -1001687848928
          │       │       │      └ Links(forward_channel_id=-1001687848928, zalets_channel_id=-1001730947390, reviews_channel_url='https://t.me/+U9ls4-OkejowMWV...
          │       │       └ Config(tg=Telegram(token='7774045102:AAGxE103tpOlQKWOW3IVPbopmk6yCGzOJMI', admin_id=52138418992), links=Links(forward_channel...
          │       └ <function Message.forward at 0x000001BB00F494E0>
          └ <Message {"message_id": 78, "from": {"id": 6948741611, "is_bot": false, "first_name": "venv∆", "username": "venvfuck", "langu...

  File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\types\message.py", line 2876, in forward
    return await self.bot.forward_message(
                 │    └ <property object at 0x000001BB0059C220>
                 └ <Message {"message_id": 78, "from": {"id": 6948741611, "is_bot": false, "first_name": "venv∆", "username": "venvfuck", "langu...
  File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\bot\bot.py", line 394, in forward_message
    result = await self.request(api.Methods.FORWARD_MESSAGE, payload)
                   │    │       │   │       │                └ {'chat_id': -1001687848928, 'from_chat_id': 6948741611, 'message_id': 78}
                   │    │       │   │       └ <aiogram.utils.helper.Item object at 0x000001BB00F9A250>
                   │    │       │   └ <class 'aiogram.bot.api.Methods'>
                   │    │       └ <module 'aiogram.bot.api' from 'C:\\Users\\newsp\\OneDrive\\Рабочий стол\\shopbot\\venv\\Lib\\site-packages\\aiogram\\bot\\ap...
                   │    └ <function BaseBot.request at 0x000001BB00FBE480>
                   └ <aiogram.bot.bot.Bot object at 0x000001BB01387710>
  File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\bot\base.py", line 236, in request
    return await api.make_request(await self.get_session(), self.server, self.__token, method, data, files,
                 │   │                  │    │              │    │       │             │       │     └ None
                 │   │                  │    │              │    │       │             │       └ {'chat_id': -1001687848928, 'from_chat_id': 6948741611, 'message_id': 78}
                 │   │                  │    │              │    │       │             └ 'forwardMessage'
                 │   │                  │    │              │    │       └ <aiogram.bot.bot.Bot object at 0x000001BB01387710>
                 │   │                  │    │              │    └ TelegramAPIServer(base='https://api.telegram.org/bot{token}/{method}', file='https://api.telegram.org/file/bot{token}/{path}')
                 │   │                  │    │              └ <aiogram.bot.bot.Bot object at 0x000001BB01387710>
                 │   │                  │    └ <function BaseBot.get_session at 0x000001BB00FBDC60>
                 │   │                  └ <aiogram.bot.bot.Bot object at 0x000001BB01387710>
                 │   └ <function make_request at 0x000001BB00F85300>
                 └ <module 'aiogram.bot.api' from 'C:\\Users\\newsp\\OneDrive\\Рабочий стол\\shopbot\\venv\\Lib\\site-packages\\aiogram\\bot\\ap...
  File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\bot\api.py", line 140, in make_request
    return check_result(method, response.content_type, response.status, await response.text())
           │            │       │        │             │        │             │        └ <function ClientResponse.text at 0x000001BB00243060>
           │            │       │        │             │        │             └ <ClientResponse(https://api.telegram.org/bot7774045102:AAGxE103tpOlQKWOW3IVPbopmk6yCGzOJMI/forwardMessage) [400 Bad Request]>...
           │            │       │        │             │        └ 400
           │            │       │        │             └ <ClientResponse(https://api.telegram.org/bot7774045102:AAGxE103tpOlQKWOW3IVPbopmk6yCGzOJMI/forwardMessage) [400 Bad Request]>...
           │            │       │        └ <property object at 0x000001BB00188630>
           │            │       └ <ClientResponse(https://api.telegram.org/bot7774045102:AAGxE103tpOlQKWOW3IVPbopmk6yCGzOJMI/forwardMessage) [400 Bad Request]>...
           │            └ 'forwardMessage'
           └ <function check_result at 0x000001BB00F85440>
  File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\bot\api.py", line 115, in check_result
    exceptions.BadRequest.detect(description)
    │          │          │      └ 'Bad Request: chat not found'
    │          │          └ <classmethod(<function _MatchErrorMixin.detect at 0x000001BB005794E0>)>
    │          └ <class 'aiogram.utils.exceptions.BadRequest'>
    └ <module 'aiogram.utils.exceptions' from 'C:\\Users\\newsp\\OneDrive\\Рабочий стол\\shopbot\\venv\\Lib\\site-packages\\aiogram...
  File "C:\Users\newsp\OneDrive\Рабочий стол\shopbot\venv\Lib\site-packages\aiogram\utils\exceptions.py", line 140, in detect
    raise err(cls.text or description)
          │   │   │       └ 'bad request: chat not found'
          │   │   └ None
          │   └ <class 'aiogram.utils.exceptions.BadRequest'>
          └ <class 'aiogram.utils.exceptions.ChatNotFound'>

aiogram.utils.exceptions.ChatNotFound: Chat not found
2024-10-28 20:30:09.542 | INFO     | aiogram.contrib.middlewares.logging:on_pre_process_my_chat_member:167 - Received chat member update for user [ID:6948741611]. Old state: {'user': {'id': 7774045102, 'is_bot': True, 'first_name': 'idkmeson', 'username': 'idkmeson_bot'}, 'status': 'left'} New state: {'user': {'id': 7774045102, 'is_bot': True, 'first_name': 'idkmeson', 'username': 'idkmeson_bot'}, 'status': 'member'} 
2024-10-28 20:30:09.684 | INFO     | aiogram.contrib.middlewares.logging:on_pre_process_message:36 - Received message [ID:79] in chat [group:-4595876202]
2024-10-28 20:30:09.684 | INFO     | aiogram.contrib.middlewares.logging:on_post_process_update:33 - Process update [ID:728620801]: [success] (in 1 ms)
2024-10-28 20:30:16.814 | INFO     | aiogram.contrib.middlewares.logging:on_pre_process_my_chat_member:167 - Received chat member update for user [ID:6948741611]. Old state: {'user': {'id': 7774045102, 'is_bot': True, 'first_name': 'idkmeson', 'username': 'idkmeson_bot'}, 'status': 'member'} New state: {'user': {'id': 7774045102, 'is_bot': True, 'first_name': 'idkmeson', 'username': 'idkmeson_bot'}, 'status': 'administrator', 'can_be_edited': False, 'can_manage_chat': True, 'can_change_info': True, 'can_delete_messages': True, 'can_invite_users': True, 'can_restrict_members': True, 'can_pin_messages': True, 'can_promote_members': False, 'can_manage_video_chats': True, 'can_post_stories': False, 'can_edit_stories': False, 'can_delete_stories': False, 'is_anonymous': False, 'can_manage_voice_chats': True} 
2024-10-28 20:30:23.708 | INFO     | aiogram.contrib.middlewares.logging:on_pre_process_my_chat_member:167 - Received chat member update for user [ID:6948741611]. Old state: {'user': {'id': 7774045102, 'is_bot': True, 'first_name': 'idkmeson', 'username': 'idkmeson_bot'}, 'status': 'left'} New state: {'user': {'id': 7774045102, 'is_bot': True, 'first_name': 'idkmeson', 'username': 'idkmeson_bot'}, 'status': 'member'} 
2024-10-28 20:30:23.708 | INFO     | aiogram.contrib.middlewares.logging:on_post_process_update:33 - Process update [ID:728620803]: [success] (in 1 ms)
2024-10-28 20:30:23.854 | INFO     | aiogram.contrib.middlewares.logging:on_pre_process_message:36 - Received message [ID:1] in chat [supergroup:-1002450057705]
2024-10-28 20:30:23.855 | INFO     | aiogram.contrib.middlewares.logging:on_post_process_update:33 - Process update [ID:728620804]: [success] (in 1 ms)
2024-10-28 20:30:23.855 | INFO     | aiogram.contrib.middlewares.logging:on_pre_process_message:36 - Received message [ID:80] in chat [group:-4595876202]
2024-10-28 20:30:39.921 | INFO     | tortoise:close_connections:552 - Tortoise-ORM shutdown
2024-10-28 20:30:39.922 | INFO     | aiogram.dispatcher.dispatcher:stop_polling:433 - Stop polling...
2024-10-28 20:30:39.922 | WARNING  | aiogram.utils.executor:start_polling:329 - Goodbye!
